README.md

```diff
@@ -1 +1,4 @@
-# inventor-be
\ No newline at end of file
+# inventor-be
+
+# Research documentation on Architectural Pattern
```

Comment:
Don't use heading 1 for this. It should be a sub heading.
Please do same for your other PRs